book/course/conda.md

```diff
@@ -0,0 +1,261 @@
+# Conda package manager
+
+Content from this lesson has been inspired and adapted from a number of sources including:
+- [Conda documentation](https://docs.conda.io/en/latest/)
+- [Carpentries Incubator: Introduction to conda for data scientists](https://carpentries-incubator.github.io/introduction-to-conda-for-data-scientists/)
+
+## Introduction
+
+Conda is an open source package management and environment management system that runs on multiple operating systems (Windows, Linux, macOS). It's features include:
+
+- Conda quickly installs, runs and updates packages and their dependencies. 
+- Conda easily creates, saves, loads and switches between environments on your local computer. 
+- It was created for Python programs, but it can package and distribute software for any language.
+
+Conda is a tool that helps find and install packages, but also lets you manage different software environments where you can install different configurations of packages. For example, this enables you to install different versions of Python in two separate environments without creating incompatibities in either of those projects.
+
+````{admonition} Conda, Miniconda and Anaconda
+:class: note
+
+It's common to be confused when confronted with Conda, Miniconda and Anaconda. Conda is specifically the package and environment manager tool itself. Miniconda is a distribution of Python that includes the Conda package manager and a few other core packages. Anaconda is another distribution of Python that includes the Conda package manager but also includes a number of widely used Python packages and other Anaconda features such as the [Anaconda Navigator](https://docs.anaconda.com/anaconda/navigator/). 
+
+```{image} ../assets/img/course/conda/miniconda_v_anaconda.png
+:alt: Miniconda versus Anaconda, reproduced from https://carpentries-incubator.github.io/introduction-to-conda-for-data-scientists/01-getting-started-with-conda/index.html
+:width: 400px
+:align: center
+```
+
+_Reproduced from [Introduction to Conda for Data Scientists](https://carpentries-incubator.github.io/introduction-to-conda-for-data-scientists/01-getting-started-with-conda/index.html)_
+
+````
+
+Conda is widely used across scientific computing and data science based domains due it's well populated package ecosystem and environment management capabilities. 
+
+- Conda installs prebuilt packages, which allows for installing complicated packages in one-step because someone else has built the tool with the right compilers and libraries
```

Comment:
```suggestion
- Conda installs prebuilt packages, which allows for installing complicated packages in one step because someone else has built the tool with the right compilers and libraries
```